compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -437,9 +439,12 @@ fn run_compiler(
 }
 
 // Extract output directory and file from matches.
-fn make_output(matches: &getopts::Matches) -> (Option<PathBuf>, Option<PathBuf>) {
+fn make_output(matches: &getopts::Matches) -> (Option<PathBuf>, Option<OutFileName>) {
     let odir = matches.opt_str("out-dir").map(|o| PathBuf::from(&o));
-    let ofile = matches.opt_str("o").map(|o| PathBuf::from(&o));
+    let ofile = matches.opt_str("o").map(|o| match o.as_str() {
+        "-" => OutFileName::Stdout,
```

Comment:
Is `-` already fixed or could we also use something that's more intuitive like `-o stdout`?

Reply:
In my original MCP I proposed using `-` because it's consistent with `gcc`, `clang` and other command-line utilities like `curl`. Also in the Zulip thread, people have been only discussing `-`, so I assume that using `-` has been agreed on.

Reply:
ok, `-` is fine then.